tests/test_cli.py

```diff
@@ -29,7 +27,8 @@ def open(self, *args, **kwargs):
         if str(args[0]).endswith(".bsv"):
             args = (
                 Path(
-                    f"./tests/test_data/mock_bsvs/{str(args[0]).rsplit('/', maxsplit=1)[-1]}"
+                    "./tests/test_data/mock_bsvs/",
+                    f"{str(args[0]).rsplit('/', maxsplit=1)[-1]}",
```

Comment:
nit: I think you can drop the surrounding f-string if you're breaking it off to its own Path() arg